db/migrate/20210711161725_create_shopping_cart_to_all_users.rb

```diff
@@ -0,0 +1,16 @@
+class CreateShoppingCartToAllUsers < ActiveRecord::Migration[6.1]
+  def change
+    reversible do |dir|
+      dir.up do
+        User.all.find_each do |u|
+          u.create_shopping_cart if u.shopping_cart.nil?
```

Comment:
to jest codesmell troche - imo tak maloznaczaca funkcjonalnosc jaka jest koszyk nie powinna wymagac tak duzych zmian w appce.